.changeset/itchy-lions-provide.md

```diff
@@ -0,0 +1,6 @@
+---
+"@chakra-ui/popper": patch
+"@chakra-ui/utils": patch
```

Comment:
```suggestion
"@chakra-ui/popper": minor
```

This should be minor since it's adding new functionality to the hook rather than fixing a bug like the component packages. 

I'm gonna create a new changeset for `@chakra-ui/utils` because it should be more detailed about what new functions were added.